### ISSUE TITLE
Range operator output throws exception in pipeline

### ISSUE THREAD

**Issue by kvprasoon** [PowerShell/PowerShell#6505]
Steps to reproduce
------------------

```
'a'..'b'|%{$_}
```

Expected behavior
-----------------

```
a
b
```

Actual behavior
---------------

```
Cannot convert value "a" to type "System.Int32". Error: "Input string was not in a correct format."
At line:1 char:1
+ 'a'..'b'|%{$_}
+ ~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidArgument: (:) [], RuntimeException
+ FullyQualifiedErrorId : InvalidCastFromStringToInteger
```

Environment data
----------------

```
> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      6.0.2
PSEdition                      Core
GitCommitId                    v6.0.2
OS                             Microsoft Windows 10.0.10586
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

Note: This works in below way.

('a'..'b')|%{$_}

**Comment by stknohg** [comment]
same issue as #5519.
This is fixed in v6.1.0-preview.1.

**Comment by iSazonov** [comment]
Leave a comment
